inc/commonitilobject.class.php

```diff
@@ -8239,4 +8239,31 @@ public static function getOpenCriteria(): array {
          ]
       ];
    }
+
+   public function displayHiddenItemsIdInput(array $options): void {
+      foreach ($options['items_id'] ?? [] as $itemtype => $items) {
```

Comment:
If `$options['items_id']` is equal to `'0'`, it will try to do a foreach on a string.
I guess you should keep old fashionned code that check for emptyness `if (!empty($options['items_id'])) { } `.

Reply:
Done in c962154.